src/modules/integrator-guide/pages/authentication-security/auth.adoc

```diff
@@ -113,30 +114,32 @@ Prohibited roles allow you to specify what roles the user is *not allowed to hav
 Forced roles allow you to specify what roles the user *must have* when accessing the {PRODUCT_NAME} API resources from a specific client.
 If the user does not have a forced role, it will be automatically added.
 |===
+//Nika's note: I don't see the forced roles in the table in the preview section.
+//line 115; specify what 'it' is for the sake of clarity.
```

Comment:
Go ahead, no need for my permission.

`the will be automatically added...` should be fine?